robottelo/ssh.py

```diff
@@ -13,6 +15,10 @@
 logger = logging.getLogger(__name__)
 
 
+class SSHCommandTimeoutError(Exception):
+    """Raised when the SSH command timeout."""
```

Comment:
I'd rephrase it, as there're few possible timeouts. `...when executed SSH command is not returning any results for certain period of time` maybe?